packages/calcite-components/src/components/accordion/accordion.tsx

```diff
@@ -117,8 +135,26 @@ export class Accordion {
   //
   //--------------------------------------------------------------------------
 
-  /** created list of Accordion items */
-  private items = [];
+  mutationObserver = createObserver("mutation", (mutationList) => {
+    let addedAccordionItem: HTMLCalciteAccordionItemElement;
+    for (const mutation of mutationList) {
+      if (mutation.type === "childList") {
+        const addedNodes = Array.from(mutation.addedNodes);
+        addedAccordionItem = addedNodes.find(
+          (node) => node instanceof HTMLCalciteAccordionItemElement
+        ) as HTMLCalciteAccordionItemElement;
+      }
+    }
+    const childNodes = Array.from(this.el.childNodes);
+    const position = childNodes.indexOf(addedAccordionItem);
```

Comment:
Do you still need `position` and sort by it after setting up the mutation observer? I think this could be simplified by updating accordion items by their order in the DOM.